bridge/eth/client.go

```diff
@@ -310,8 +328,15 @@ func (c *Client) finish(auth *bind.TransactOpts, signatures [][]byte, batch *bri
 	return c.bridgeContract.FinishCurrentPendingBatch(auth, batch.Id, proposedStatuses, signatures)
 }
 
-func (c *Client) SignersCount(context.Context, bridge.ActionId) uint {
-	return uint(len(c.broadcaster.Signatures()))
+func (c *Client) SignersCount(_ context.Context, batch *bridge.Batch, actionId bridge.ActionId) uint {
```

Comment:
add comment

Reply:
maybe to all exported stuff in this file

Reply:
done